src/compiler/backports.md

```diff
@@ -0,0 +1,161 @@
+team members who are subscribed to the backport channel.
+
+Compiler team members will review the beta/stable backport nominations through the zulip threads,
+and discuss and vote on the polls created by [triagebot](../triagebot/zulip-notifications.md). If
+there are no timely consensus on decision from the asynchronous backport nomination zulip threads,
+the [compiler-ops team](../compiler/operations.md) will raise those on the upcoming [compiler triage
+meeting](./meetings.md) to make a synchronous decision.
+
+### Considerations for backport decisions
+
+For compiler backport reviewers, here are some **non-exhaustive** considerations that they might
+consider in making a backport decision:
+
+- How long does a beta backport have to bake? Consider remaining time available for users to test
+  the beta compiler candidate before the next release cycle (note that the actual stable compiler
+  artifact will necessarily be built sooner than the actual release!).
+- How complex is the nominated compiler change?
+- How risky is the nominated compiler change? Can it introduce new regressions to the stable/beta
+  channel? Can the "cure" (nominated change) be potentially worse than the "poison" (the stable/beta
+  regression)?
+- How severe is the regression/issue being fixed? E.g. critical regressions vs stable-to-stable
+  regressions that are not very severe, and is already on stable channel for multiple release
+  cycles.
+
+At the end of then day, it's a risk vs benefit tradeoff decision. The general guidance is to only
+accept beta backports if it's fix for a (beta/stable) regression if the benefit sufficiently
+overweighs the risk. For stable backports, this bar should be even higher as we should not and do
+not want to encourage frequent stable point releases.
+
+## When and how is a backport decision reached? How long should a backport decision take?
+
+> **Overall idea**
+>
+> The general idea behind the current async backport review and approval process experiment is that
+>
+> > async backport will never take longer than waiting for the next meeting.
+>
+> See [discussions on async backport reviews](https://rust-lang.zulipchat.com/#narrow/channel/131828-t-compiler/topic/Do.20backport.20nominations.20async.3F/near/518255539).
+
+A *asynchronous* beta/stable backport nomination (i.e. the aforementioned async backport nomination
+zulip threads under [#t-compiler/backports]) **may** be considered accepted/rejected **if and only
+if**:
+
+- There are at least **3** compiler team members who vote for approve/reject (i.e. a "**strong
+  decision outcome**"), and there are no outstanding votes in the opposite direction, then the
+  backport nomination can be considered to be approved/accepted **by the next compiler triage
+  meeting**.
+  - Example: if **3** compiler team members vote **approve**, **1** member vote **don't know**, then
+    the backport nomination is considered accepted by the next compiler triage meeting.
```

Comment:
Discussion:

1. Is `N = 3` a good minimum threshold for async backport polls?
2. Does the `N = 3` need to include `>= 1` compiler lead?